src/canisters/frontend/ic-asset/src/canister_api/types/batch_upload/v1.rs

```diff
@@ -0,0 +1,41 @@
+use crate::canister_api::types::{
+    asset::SetAssetPropertiesArguments,
+    batch_upload::common::{
+        ClearArguments, CreateAssetArguments, DeleteAssetArguments, SetAssetContentArguments,
+        UnsetAssetContentArguments,
+    },
+};
+use candid::{CandidType, Nat};
+
+/// Batch operations that can be applied atomically.
+#[derive(CandidType, Debug, Eq, PartialEq, PartialOrd, Ord)]
+pub enum BatchOperationKind {
+    /// Create a new asset.
```

Comment:
Please see https://github.com/dfinity/sdk/pull/3085 and reorder these accordingly

Reply:
I just learned that when ord is derived for enum, variants are ordered by their top-to-bottom discriminant order.
rust is so awesome